apps/api/src/lambda.ts

```diff
@@ -1,32 +1,21 @@
-import { NestFactory } from '@nestjs/core';
 import serverlessExpress from '@vendia/serverless-express';
-import express from 'express';
 import {
   APIGatewayProxyEvent,
   APIGatewayProxyResult,
   Context,
   Callback,
   Handler,
 } from 'aws-lambda';
-import { AppModule } from './app.module';
-import { ExpressAdapter } from '@nestjs/platform-express';
-import { API_PREFIX } from './config';
+import { createNestApp } from './main';
 
 let cachedServer: Handler;
 
 async function bootstrap() {
   if (!cachedServer) {
-    const expressApp = express();
-
-    const nestApp = await NestFactory.create(AppModule, new ExpressAdapter(expressApp));
-
-    nestApp.setGlobalPrefix(API_PREFIX);
-
+    const { app: nestApp, expressApp } = await createNestApp();
     await nestApp.init();
-
     cachedServer = serverlessExpress({ app: expressApp });
   }
-
```

Comment:
@fwpushan have you tested if this is working locally?

Reply:
regular app is working locally.